app/uk/gov/hmrc/incorporatedentityidentificationfrontend/models/JourneyConfig.scala

```diff
@@ -30,7 +31,7 @@ object JourneyConfig {
       continueUrl <- (json \ continueUrlKey).validate[String]
       optServiceName <- (json \ optServiceNameKey).validateOpt[String]
       deskProServiceId <- (json \ deskProServiceIdKey).validate[String]
```

Comment:
signOutUrl <- (json \ signOutUrlKey).validate[String] should be the signOutUrl your passing in to line 34